tools/detect_unused_modules

```diff
@@ -1,9 +1,11 @@
 #!/bin/sh -e
 
-for test in $(git ls-files "*tests/*.pm") ; do
+MODULES="${@}"
+
+for test in $MODULES ; do
     t=$(echo $test | sed -e 's@tests/.*/@@' -e 's@\.pm$@@')
     t_dir=$(echo $test | sed -e 's@tests/\(.*\)/.*$@\1@')
-    git grep -E "(- $t_dir/$t|loadtest.*$t|load_testdir.*$t_dir|^use (base )?\"?$t\"?)" | grep -qv ".(pm|yaml):\s*#" || echo $test
+    git grep -E "(- $t_dir/$t|loadtest.*$t|load_testdir.*$t_dir|^use (base )?\"?$t\"?)" | grep -qv ".(pm|yaml):\s*#" || echo "$test module is not used in any schedule"
```

Comment:
How is old line is better? It gives you the name of the module and that's it. So you will have to find the line to figure out which script was executed. I'm not going to change that, old message is obscure.

Reply:
The old line is better in just outputting the unused modules so the output can be directly piped to any other command call. Embedding the file name in a longer message can certainly be helpful but if you need it then I suggest to do that only in a requested mode, e.g. verbose mode

Reply:
I doubt that anyone uses it this way or will ever use, but when CI fails everyone will spend additional time on investigation.

Reply:
> The old line is better in just outputting the unused modules so the output can be directly piped to any other command call. Embedding the file name in a longer message can certainly be helpful but if you need it then I suggest to do that only in a requested mode, e.g. verbose mode

Exactly because of Rodion's hunch on anybody using it ever, but only looking at the CI when it fails, it's going to be easier to find a filename + a message, than a filename saying nothing. An implicit expectation that reviewer will read line by line and understand what a lone filename means, is a bit too optimistic.